string object_name
---
geometry_msgs/Point location
